README.md

```diff
@@ -168,3 +190,72 @@ targets:
     dependencies:
       - "barback"
 ```
+
+#### Defining `Builder`s in your package (similar to transformers)
+
+If users of your package need to apply some code generation to their package,
+then you can define `Builder`s (from `package:build`) and have those be either
+be automatically applied based on existing transformer settings,or simply
+available for users to opt into as needed.
+
+You tell bazelify about your `Builder`s using the `builders` section of your
+`bazelify.yaml`. This is a map of builder names to configuration. Each builder
+config may contain the following keys:
+
+- **target**: The name of the target which defines contains your `Builder` class
+  definition.
+- **import**: Required. The import uri that should be used to import the library
+  containing the `Builder` class. This should always be a `package:` uri.
+- **class**: The name of the `Builder` class to instantiate, must be exported by
+  the library referenced by `import`.
+  - We could do what transformers do and use mirrors/codegen to just find these,
```

Comment:
I'd keep this in the commit message rather than in the readme. End users don't necessarily need this information